csharp/AppEncryption/AppEncryption/Persistence/DynamoDbMetastoreImpl.cs

```diff
@@ -167,6 +203,19 @@ public Option<JObject> LoadLatest(string keyId)
             }
         }
 
+        /// <summary>
+        /// Stores the <see cref="JObject"/> value using the specified keyId and created time. In case of GlobalTables,
+        /// the key needs to be formed using the <see cref="DynamoDbMetastoreImpl.GetHashKey"/> method, which may or may
+        /// not add a region suffix to it.
```

Comment:
For `Store` and `LoadLatest` the final key to use _always_ comes from the `GetHashKey` method